examples/platform/linux/Options.cpp

```diff
@@ -383,8 +394,21 @@ const char * sDeviceOptionHelp =
     "  --faults <fault-string,...>\n"
     "       Inject specified fault(s) at runtime.\n"
 #endif
-    "   --dac_provider <filepath>\n"
+    "  --dac_provider <filepath>\n"
     "       A json file with data used by the example dac provider to validate device attestation procedure.\n"
+#if CHIP_CONFIG_ENABLE_ICD_SERVER
+    "  --icdActiveModeDurationMs <icdActiveModeDurationMs>\n"
+    "       Sets the ICD active mode threshold (in milliseconds). (Default: 300) \n"
+    "       This defines the how long the the server typically will stay in active mode after \n"
+    "       initial transition out of idle mode.\n"
+    "  --icdIdleModeDuration <icdIdleModeDuration>\n"
+    "       Sets the ICD idle mode durations (in seconds). (Default: 300) \n"
+    "       This defines the how long the ICD server can stay in idle mode.\n"
+    "  --icdPeriodicWakeupDurationMs <icdPeriodicWakupDurationMs>\n"
+    "       Sets the ICD periodic wakeup (in milliseconds). (Default: 60000)\n"
+    "       This is for testing purpose on Linux which defines the internal to wakeup periodically to send \n"
+    "       Check-In messages since Linux application doesn't have a physical button to trigger wakeup\n"
```

Comment:
The button to trigger wake should not be simulated periodically.

Also please clarify between LIT ICD and SIT ICD. The features enabled here are LIT ICD specific for our purposes.

Reply:
--icdPeriodicWakeupDurationMs removed. Replace it with RPC